core/modules/image/image.module

```diff
@@ -962,6 +963,9 @@ function image_style_flush($style) {
  * @see image_style_deliver()
  */
 function image_style_url($style_name, $uri) {
+  if (image_is_svg($uri)) {
+    return file_create_url($uri);
```

Comment:
Hm. This return happens before the clean_url check below. Isn't that problematic?

Reply:
Hm, I don't know this section yet.

Reply:
I think that's okay since it seems that the clean_url check is only if it needs to run through an image derivative callback. Maybe. (I'm now questioning whether want to skip image styles altogether. It's a bit confusing that users can't rotate, crop or do any other effects on the SVG. The svg_image module in D7 does this:

```
$element[$delta] = array(
      '#theme' => 'image_formatter',
      '#item' => $item,
      '#image_style' => $is_svg === FALSE ? $display['settings']['image_style'] : '',
      '#path' => isset($uri) ? $uri : '',
    );
```

And skips `image_style_url()`.

Reply:
I added a comment with a potential solution to image-style partial SVG support in the parent issue: Perhaps just add a note when something won't work as expected for SVGs?